src/mapping/tests/AxialGeoMultiscaleMappingTest.cpp

```diff
@@ -243,3 +243,93 @@ BOOST_AUTO_TEST_CASE(ConsistentCollectZ)
 
 BOOST_AUTO_TEST_SUITE_END()
 BOOST_AUTO_TEST_SUITE_END()
+
+PRECICE_TEST_SETUP(1_rank);
+BOOST_AUTO_TEST_CASE(ConsistentSpreadXScalar)
+{
+  PRECICE_TEST();
+  // 1D -> 3D, scalar data: the same value is copied to all output vertices, along X axis
```

Comment:
This case needs to be updated to consider the spread profile, as this one would be uniform: https://github.com/precice/precice/pull/2386

When that gets merged, the unique characteristic of this tutorial would be the scalar data, not that the same value is copied to all output vertices. The comment could better reflect this:


```suggestion
  // 1D -> 3D, scalar data, uniform spread along the X axis
```